src/auth/auth-fetch.ts

```diff
@@ -70,3 +63,6 @@ export const withRetryAfterReloadingExpiredAuthToken =
       }
     }
   };
+
+export const authOpts = _authOpts;
+export const withAuthSession = _withAuthSession;
```

Comment:
this re-export is a good middle-ground to keep things low-impact for most consumers that don't have the circular dependency trap to worry about.